components/learn/modules/ROOT/pages/public-staging.adoc

```diff
@@ -84,7 +84,14 @@ const HDWalletProvider = require('@truffle/hdwallet-provider');
 
 module.exports = {
   networks: {
-    development: { ... },
+    development: {
```

Comment:
I guess the intention here is to let users copy-paste the code into their own network file, but that may not be the best idea here, where we're _adding_ content and not creating a new file from scratch.

Perhaps making this a `diff` and showcasing the new lines would be better?

Reply:
Given that it is six lines I prefer to have the whole thing, as we are mostly adding content.  A diff would be better than what might appear to be a working `network.js`

When I walked through it, I copied the whole network.js as is and then had to investigate why I had errors.  

I assume that for the people who do what I did, it is worth making their lives easier over saving six lines.

Reply:
I tried both approaches and think `diff` works much better, since it's not just the object entry but also the imports of the secret, provider, etc.

Reply:
Diff is clear. Though a PITA to copy if using as a tutorial.